src/dlangui/core/collections.d

```diff
@@ -248,6 +248,17 @@ struct ObjectList(T) {
         assert(index >= 0 && index < _count, "child index out of range");
         return _list[index];
     }
+    /// get item by index. Returns null if item not found
+    inout(T) tryGet(int index) @safe inout nothrow {
+        try {
+            if (index < 0 || index >= _count) {
+                return null;
+            }
+            return _list[index];

```

Comment:
I don't think it's right to try-catch this code. Condition on top should be enough to make this never throw, so you're just silencing other potential issues here. Don't mark this as nothrow and remove try-catch